linkme-chatapp/devServer.js

```diff
@@ -68,35 +72,42 @@ function createApp() {
     if (!layerProviderID) return res.status(500).send('Couldn\'t find LAYER_PROVIDER_ID');
```

Comment:
My personal convention is to never have ifs without { }. Will also make it a bit more readable 😄